src/app/static/src/app/components/projects/ProjectHistory.vue

```diff
@@ -30,7 +30,9 @@
                     </button>
                 </div>
                 <div class="col-md-2 project-cell">
-                    {{ p.name }}
+                    <span
+                    v-on:click="loadVersion($event, p.id, v.id)">
```

Comment:
Sorry, I'm still not sure what's going on with this link - it's not actually in master. Did the branch with the load version link never get merged? Or did it get merged and something overrode it? Can you remember what ticket that was in?